common/shlibs

```diff
@@ -4049,3 +4053,4 @@ libtdb-wrap-samba4.so samba-libs-4.13.2_1
 libutil-cmdline-samba4.so samba-libs-4.13.2_1
 libwbclient.so.0 samba-libs-4.13.2_1
 libwinbind-client-samba4.so samba-libs-4.13.2_1
+libctdb-event-client-samba4.so ctdb-4.13.2_1
```

Comment:
Please try adding `ctdb-event-client` to the `_privlibs` list to see if this will build without adding another unversioned shared library that we need to track.